docs/modules/integrate/pages/kafka-connect-connectors.adoc

```diff
@@ -0,0 +1,122 @@
+:page-beta: true
+
+{description}
+
+link:https://www.confluent.io/hub/[Kafka Connect Source connectors] are available for over 100 popular platforms, including RabbitMQ, Neo4j, Couchbase, Scylla, SAP, and Redis.
+
+== Installing Connector Dependencies
+
+Kafka Connect Source connector dependencies are included in the slim and full distributions of Hazelcast.
+
+To use the latest version of the dependencies, add the `hazelcast-jet-kafka-connect` module to your Java project.
+
+[tabs] 
+====
+Maven:: 
++ 
+--
+[source,xml,subs="attributes+"]
+----
+<dependency>
+  <groupId>com.hazelcast.jet</groupId>
+  <artifactId>hazelcast-jet-kafka-connect</artifactId>
+  <version>{full-version}</version>
+  <classifier>tests</classifier>
+</dependency>
+----
+--
+Gradle:: 
++ 
+--
+[source,shell,subs="attributes+"]
+----
+compile group: 'com.hazelcast.jet', name: 'hazelcast-jet-kafka-connect', version: ${full-version}
+----
+--
+====
+
+== Downloading the Kafka Connect Source Connector
+
+Download the correct link:https://www.confluent.io/hub/[Kafka Connect Source connector] for the data source that you want to integrate. Kafka Source connectors consist of either a ZIP file that contains the connector JAR and dependencies, or a single JAR that contains everything you need. 
+
+Every Kafka Connect Source connector comes with documentation that includes the following:
+
+- Features of the connector
+- Configuration details
+
+== Permissions
+[.enterprise]*Enterprise*
+
+If xref:security:enabling-jaas.adoc[security] is enabled, your clients may need updated permissions to upload the ZIP or JAR file used by the Kafka Connect Source Connector. For details, see xref:pipelines:job-security.adoc[].
+
+== Adding the Connector Configuration
+
+To use your Kafka Connect Source connector as a streaming data source, you need to add the connector configuration to a pipeline.
+
+All examples in this section are from a pipeline that uses a Kafka-based Source connector to generate random numbers.
+
+Start by creating a `Properties` object for your pipeline.
+
+```java
+Properties props = new Properties();
+
+props.setProperty("name", "random-source-connector"); <1>
+props.setProperty("connector.class", "sasakitoa.kafka.connect.random.RandomSourceConnector");
+
+props.setProperty("tasks.max", "1"); <2>
+
+props.setProperty("generator.class", "sasakitoa.kafka.connect.random.generator.RandomInt"); <3>
+props.setProperty("messages.per.second", "1000");
+props.setProperty("topic", "test");
+props.setProperty("task.summary.enable", "true");
+```
+
+<1> Mandatory properties: Only the unique connector `name` and `connector class` are required.
+<2> Common properties: You can use any of the link:[common properties] of Kafka Connect in your pipeline depending on your use case.
```

Comment:
`link:[common properties]` seems to be broken

Reply:
Fixed.

Reply:
Closed.